test/sharness/t0110-gateway.sh

```diff
@@ -63,6 +65,19 @@ test_expect_success "GET invalid path errors" '
   test_must_fail wget http://127.0.0.1:5002/12345
 '
 
+
+test_expect_success "GET webui resolves" '
+  wget http://127.0.0.1:5002/ipfs/$webui_hash -O expected
+  wget "http://127.0.0.1:5001/webui" -O actual1
+  wget "http://127.0.0.1:5001/webui/" -O actual2
+  test_cmp expected actual1 &&
+  test_cmp expected actual2 &&
```

Comment:
not sure if this will work in all cases, because the webui isn't shipped with go-ipfs directly.